.storybook/main.ts

```diff
@@ -25,14 +29,10 @@ const storybookConfig: StorybookConfig = {
     '../src/**/*.mdx',
   ],
   addons: [
-    '@storybook/addon-backgrounds',
-    '@storybook/addon-a11y',
-    '@storybook/addon-viewport',
-    '@storybook/addon-controls',
-    '@storybook/addon-toolbars',
-    './SidebarLabel/preset.cjs',
+    getAbsolutePath('@storybook/addon-a11y'),
+    // getAbsolutePath('./SidebarLabel/preset.cjs'),
```

Comment:
Should we put this back now?
```suggestion
    getAbsolutePath('./SidebarLabel/preset.cjs'),
```

Reply:
Relates to my comment above. Storybook doesn't start when its enabled, but looking at v8 where it is enabled, I can't see it in action. Not sure what it actually does.

Reply:
Oops, didn't see your comment! If it's not used, then should we delete it?

Reply:
Think we can, just more of a question if anyone knew if it is actually being used in our current v8 and I've missed it!

Reply:
I think we "copied" it from webcore's storybook & we were going to display component health in the sidebar. It doesn't appear to be in use anymore, so I think we can just delete it.

Reply:
Cool, removed!

Reply:
Great! When this is merged, are you also OK to close the JIRA ticket (I added it to the backlog due to allll of the failing tests - thanks for fixing!)

Reply:
Ah! Yes I will do. I did a quick search and couldn't see it in the tech backlog, but just realised its because I had a filter on 🤦

Reply:
It's also linked at the top of the PR 👍

Reply:
Doh, missed the first line lol. My eyes immediately focus on the dependency table in these type of PRs.